templates/desktop/developers.html

```diff
@@ -69,7 +70,9 @@ <h2>Security support <br aria-hidden/>for Ubuntu releases</h2>
   </div>
   <div class="u-fixed-width">
     <div class="u-hide--small" id="server-desktop-eol"></div>
-    {% include "about/release_cycles/releases-table.html" %}
+    <div class="u-hide--large u-hide--medium">
```

Comment:
@carkod It's supposed to show on the small screen, right?

Reply:
Yes. The graph will become a table in small screens (that functionality was already there) and the table that we are huding here should show

Reply:
aha right, it's just that the QA process says that the mobile doesn't show the table so I got confused!

Reply:
Sorry my bad, I meant the complete opposite hah 😅

Reply:
haha that's what I thought :) no worries at all